### ISSUE TITLE
This repo needs to be archived and marked as deprecated

### ISSUE THREAD

**Issue by spencerbart** [fitzgen/wee_alloc#109]
Too many sites are recommending this repo but this repo has major security issues. It needs to be fixed or taken down.

**Comment by erwanvivien** [comment]
Agreed

**Comment by jack-mil** [comment]
Yes I am beginning to see that a lot of the the Rust wasm ecosystem is outdated and not working well in 2023. 
Specifically the tutorials and book provided from https://rustwasm.github.io/ has not been updated for years and does not work out of the box.
What is the modern tooling for wasm with rust, besides wasm-pack and friends?

**Comment by FlogramMatt** [comment]
Major security issue? You mean the memory leak?  You do realize it's impossible to free wasm memory.. I would say this is a wasm issue not this library. I bet the standard allocator has the same issue..

**Comment by erwanvivien** [comment]
https://github.com/rustwasm/wee_alloc/issues/106#issue-1179853771

> Making two large allocations, then dropping them in the order they were allocated leaks memory in wee_alloc, but not in default allocator.

Hello @FlogramMatt. I believe not, this issue says otherwise
